test/DebugInfo/DebugInfo-GV-with-SMD-and-DIE.ll

```diff
@@ -0,0 +1,41 @@
+;; Ensure that DIExpressions are preserved in DIGlobalVariableExpressions
```

Comment:
same comments are applicable to this test as well

Reply:
Added checking for nonsemantic-shader-100.  Added SPV testing.